scripts/patch.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This file will require changing Dockerfile in #183. Maybe we should merge this one first.

Reply:
Ack